docs/operate-and-deploy/installation/server-config/security.md

```diff
@@ -219,6 +219,134 @@ credentials when starting the CLI by using the `--user` and
 <ksql-install>bin/ksql --user fred --password letmein http://localhost:8088
 ```
 
+Configure ksqlDB for Internal Authentication
```

Comment:
> I've just had a look at what's already in this document and I think, rather than adding a brand new section, the details you're covering need to be integrated with what is already there.

I followed your instructions fairly closely, so I think we're good there.

@JimGalasyn Can you take a look at my docs changes again?  Also, How can I verify that the local anchor links work correctly?  None of my tools allow for checking this.